lib/adal.js

```diff
@@ -316,6 +327,12 @@ AuthenticationContext.prototype._renewIdToken = function (callback) {
     this._loadFrame(urlNavigate, 'adalIdTokenFrame');
 };
 
+AuthenticationContext.prototype._urlContainsQueryStringParameter = function (name, url) {
+    name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+    var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
```

Comment:
@RandalliLama ... Thank you for the response. I'll look at simplifying that, and re-submit an update.